### ISSUE TITLE
I am wondering if you can share how to publish a trained model in the huggingface. Thank you! 

### ISSUE THREAD

**Issue by kevin-liu-astrumu** [machamp-nlp/machamp#32]

**Comment by robvanderg** [comment]
The AutoModel can be extracted with the following code:

`from machamp.model.machamp import MachampModel
import torch

path = 'logs/ewt/2023.05.09_12.12.25/model.pt'

model = torch.load(path)
autoModel = model.mlm
` 
This can be ran from the root directory of your machamp installation.

From here, the standard huggingface methods should work (you might want to add the matching tokenizer). However, it would be nice to have some cleaner standalone script for this in the future.

It should be noted that MaChAmp is based on the AutoModel class, and uploading the classification heads would be non-trivial (the above code only loads the encoder). One would first have to initiate a specific model from huggingface, and then copy over the weights into the right variable manually, while making sure the indices of the labels remain the same.

**Comment by robvanderg** [comment]
We actually included a more in-depth script that I forgot about; usage here: https://github.com/machamp-nlp/machamp/blob/master/docs/extract_model.md
